src/navigation/utilities.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2020
+ */
+
+import { RootNavigator } from './RootNavigator';
+import { goBack } from './actions';
+import { ROUTES } from './constants';
+
+import { navStrings } from '../localization';
+
+export const SCREEN_TITLES = {
```

Comment:
I don't like this for getting a screens title - open to suggestions!

Reply:
Hmm, I don't think this is too bad? Can't really think of another way of doing it that's any tidier tbh.